<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dendrite</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>421</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>30</horstretch>
      <verstretch>30</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Upload</string>
     </attribute>
     <widget class="QToolButton" name="uploadFile">
      <property name="geometry">
       <rect>
        <x>17</x>
        <y>60</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose File</string>
      </property>
     </widget>
     <widget class="QFrame" name="previewPhoto">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>221</width>
        <height>161</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="UploadFileName">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>62</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>tool for proving spatial data transfer between cells</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>144</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Automatic identification of parallel growth among neuronal dendritic branches</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Properties</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Edges</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>19</y>
         <width>141</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>threshold1:</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>20</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_2">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>50</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>250</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>49</y>
         <width>131</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>threshold2:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>401</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Merge Lines</string>
      </property>
      <widget class="QSlider" name="horizontalSlider_3">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>19</y>
         <width>281</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min distance to merge:</string>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_4">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>191</width>
         <height>31</height>
        </rect>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="tracking">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>49</y>
         <width>281</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>min angle to merge:</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Results</string>
      </property>
      <widget class="QComboBox" name="Units">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>161</width>
         <height>39</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Degree</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Radian</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>301</width>
         <height>33</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="text">
        <string>angle units:</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="OkButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>270</y>
        <width>144</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
